bin/enqueue.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+if [ ! -f DeepSpeech.py ]; then
+  echo "Please make sure you run this from DeepSpeech's top level directory."
+  exit 1
+fi
+
+GLOBAL_LOG="\\/dev\\/null" # by default there is no global logging
+NODES=2 # to be adjusted
+GPUS=8 # to be adjusted
```

Comment:
They are default values for command line parameters. Auto-detecting them would be complicated and error-prone. They also represent default values that don't have to represent the cluster's maximum topology, but just a (to be defined) fraction of it.